app.json

```diff
@@ -2,7 +2,7 @@
   "name": "Mastodon",
   "description": "A GNU Social-compatible microblogging server",
   "repository": "https://github.com/tootsuite/mastodon",
-  "logo": "https://github.com/tootsuite/mastodon/raw/master/app/javascript/images/logo.svg",
```

Comment:
The raw format of GitHub returns `Content-type: text/plain`. SVG will not be rendered unless `Content-type` is `image/svg+xml` when imported with `img` element.

I thought it would be better to use an avatar image with a permanently URL.